datanode/gateway/graphql/schema.graphql

```diff
@@ -2206,6 +2238,18 @@ type PositionUpdate {
   positionStatus: PositionStatus!
 }
 
+"Basic description of an order"
+input OrderInfo {
+  "Whether the order is to buy or sell"
+  side: Side!
+  "Price for the order"
+  price: String!
+  "Number of units remaining of the total that have not yet been bought or sold (uint64)"
+  remaining: String!
+  "Boolean indicating a market order"
+  isMarketOrder: Boolean
```

Comment:
is this truly Boolean or undefined? Update to `Boolean!` if not

Reply:
I thought an optional boolean (assume false if not specified) will be more user friendly, but perhaps it might just lead to confusion.

Reply:
I'd prefer a stricter type, but consider this non blocking

Reply:
Fair enough, changed now.